products.css

```diff
@@ -0,0 +1,99 @@
+@import url(/global-styles.css);
+.nav-bar{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0.5rem 1.5rem;    
```

Comment:
Use px for sizes less than 1rem because scaling such a small value in rem is not generally desirable.